rspec-expectations/lib/rspec/matchers/built_in/change.rb

```diff
@@ -326,8 +326,8 @@ def from(value)
 
         # @private
         def does_not_match?(_event_proc)
-          raise NotImplementedError, "`expect { }.not_to change { }.to()` " \
-                                     "is not supported"
+          raise NotImplementedError, "Using `expect { }.not_to change { }` " \
```

Comment:
We've changed the failure messages, but no specs failed.
May I kindly ask you to add such specs for the `.to()` and `.by()` cases?

Reply:
Yeah, I can do that. I'll wait for this to merge first?
https://github.com/rspec/rspec/pull/178

Reply:
It’s fine, I’ll adjust and rebase that other pr after yours. With specs for error messages this will be an easy thing to do.